src/main/java/org/apache/flink/graph/streaming/library/BipartitenessCheck.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.graph.streaming.library;
+
+import org.apache.flink.api.common.functions.GroupReduceFunction;
+import org.apache.flink.api.common.functions.ReduceFunction;
+import org.apache.flink.graph.streaming.EdgesFold;
+import org.apache.flink.graph.streaming.WindowGraphAggregation;
+import org.apache.flink.graph.streaming.example.util.Candidates;
+import org.apache.flink.graph.streaming.example.util.SignedVertex;
+import org.apache.flink.types.NullValue;
+
+import java.io.Serializable;
+
+/**
+ * The bipartiteness check library to check whether an input graph is bipartite
+ * or not. A bipartite graph's vertices can be separated into two disjoint
+ * groups, such as no two nodes inside the same group is connected by an edge.
+ * The library uses the merge-tree abstraction of our graph streaming API.
```

Comment:
changed it to:
The library uses the Window Graph Aggregation class of our graph streaming API.